src/apps/replication/client_ddl_tool/main.cpp

```diff
@@ -0,0 +1,102 @@
+#include <dsn/service_api_c.h>
+#include <dsn/ports.h>
+#include <dsn/dist/replication/empty_app.h>
+#include <dsn/dist/replication/client_ddl.h>
+#include <iostream>
+
+using namespace dsn::client;
+
+void usage(char* exe)
+{
+    std::cout << "Usage:" << std::endl;
+    std::cout << "\t" << exe << " <config.ini> create_app -name <app_name> -type <app_type> [-pc partition_count] [-rc replication_count]" << std::endl;
+    std::cout << "\t" << exe << " <config.ini> drop_app -name <app_name>" << std::endl;
```

Comment:
sure,
list_app to list all apps of these info: [app_name] [app_id] [app_type] [partition_count] [replica_count]
desc_app to describe a special app, including partition_config of all partitions.